modules/browsemap/src/main/java/com/sygic/modules/browsemap/BrowseMapFragment.kt

```diff
@@ -109,15 +111,30 @@ class BrowseMapFragment : MapFragmentWrapper() {
         return root
     }
 
+    fun addOnMapClickListener(onMapClickListener: (poiData: PoiData) -> Unit) {
```

Comment:
ked to pridas cez tuto funkciu, tak uz nikdy nebudes vediet zavolat remove nie?

Reply:
pravdu máš. přidal jsem jí sem jen proto, aby mi šla lambda v kotlinu. můžeme na to ještě juknout ;)